### DISCUSSION TITLE
Don't offer completions for imports that are not installed

### DISCUSSION THREAD

**Discussion by codebysandwich** [microsoft/pylance-release#5623] [Enhancement]
for example

package `pandas` install in system env not installed in virtual env

however autocomplete get the pandas completed, set the interpreter correctly. checked pip list in terminal. 

<img width="1737" alt="image" src="https://github.com/microsoft/pylance-release/assets/32191093/e33808c2-b679-462c-8715-92088ecd3bc4">
 
autocompleted but not the right way
<img width="1367" alt="image" src="https://github.com/microsoft/pylance-release/assets/32191093/de77b0d6-ee4a-479a-8ce9-74df1e2acbbf">

**Comment by StellaHuang95** [comment]
Thanks for reporting this issue. This is by design. Please see this comment for why we are doing this. https://github.com/microsoft/pylance-release/issues/2016#issuecomment-955376007. Do you see other issues with this behavior?

**Comment by codebysandwich** [comment]
> Thanks for reporting this issue. This is by design. Please see this comment for why we are doing this. [#2016 (comment)](https://github.com/microsoft/pylance-release/issues/2016#issuecomment-955376007). Do you see other issues with this behavior?

thanks your answer!, could figure out the design ,whether provided a setting in setting.json disable this feature?

**Comment by StellaHuang95** [comment]
Are you asking if there's a way to disable autocompletion entirely or specifically excluding 'pandas' from the completion list? There isn't a way to just disable `pandas`, but you can disable autocompletion by adding the following to your `settings.json`. 

```json
"editor.quickSuggestions": {
  "other": "off",
  "comments": "off",
  "strings": "off"
}
```

**Comment by StellaHuang95** [comment]
This is a reasonable ask though, let me move this issue to discussion as an enhancement request for comments and upvotes.
